meta-lmp-base/recipes-support/lmp-device-auto-register/lmp-device-auto-register.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "A systemd oneshot helper to auto register a device"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+RDEPENDS_lmp-device-auto-register = "lmp-device-register"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+SRC_URI = " \
+	file://lmp-device-auto-register.service \
+	file://lmp-device-auto-register \
+	file://api-token \
```

Comment:
Where this token comes from? Is it going to be hardcoded into meta-lmp and it will be shared across all factories?

Reply:
This is empty by default, a factory can have their own token by doing something like:

```
diff --git a/recipes-support/lmp-device-auto-register/lmp-device-auto-register.bbappend b/recipes-support/lmp-device-auto-register/lmp-device-auto-register.bbappend
new file mode 100644
index 0000000..72d991c
--- /dev/null
+++ b/recipes-support/lmp-device-auto-register/lmp-device-auto-register.bbappend
@@ -0,0 +1 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
diff --git a/recipes-support/lmp-device-auto-register/lmp-device-auto-register/api-token b/recipes-support/lmp-device-auto-register/lmp-device-auto-register/api-token
new file mode 100644
index 0000000..ba55264
--- /dev/null
+++ b/recipes-support/lmp-device-auto-register/lmp-device-auto-register/api-token
@@ -0,0 +1 @@
+TOKEN
```